src/resources/js/app.js

```diff
@@ -12,9 +13,10 @@ if (process.env.NODE_ENV === "development") {
 }
 
 
-new Vue({
+const app = new Vue({
```

Comment:
Do you really need to declare a `const` ? It isn't used anywhere.

This will generate a linting error and you'll be big sad

Reply:
is ok now